app/assets/javascripts/modules/track_search.js

```diff
@@ -0,0 +1,24 @@
+(function(Modules) {
+  "use strict";
+
+  Modules.TrackSearch = function() {
+    var that = this;
+
+    that.start = function(container) {
+
+      var filterSearchInput = container.find('table'),
+          tableInput = container.find('.js-filter-table-input');
+      filterSearchInput.on('keyup', '.js-filter-table-input', trackSearch);
```

Comment:
Ideally it's to send what's being searched back to GA. However, we could just fire that the filter was being used. 

I'll wait to talk to Tim LB before continuing with this, as it might be dropped.